`test_torch_sigmoid_approx` is flaky

**Describe the bug**
The `test_torch_sigmoid_approx` test fails intermittently during automated PR testing.

**To Reproduce**
Run the test (or full suite) until it fails.

**Expected behavior**
The test should reliably pass.

**Screenshots**
```
____________________ test_torch_sigmoid_approx[exp-3-0.06] _____________________
[...]
    def test_torch_sigmoid_approx(method, prec_frac, tolerance, workers):
        """
        Test the approximate sigmoid with different tolerance depending on
        the precision_fractional considered
        """
        alice, bob, james = workers["alice"], workers["bob"], workers["james"]
    
        t = torch.tensor(range(-10, 10)) * 0.5
        t_sh = t.fix_precision(precision_fractional=prec_frac).share(alice, bob, crypto_provider=james)
        r_sh = t_sh.sigmoid(method=method)
        r = r_sh.get().float_prec()
        t = t.sigmoid()
        diff = (r - t).abs().max()
        norm = (r + t).abs().max() / 2
    
>       assert (diff / (tolerance * norm)) < 1
E       assert (tensor(0.0605) / (0.06 * tensor(0.9890))) < 1
```

**Additional context**
This started being an issue when the order of the test suite was randomized (specifically to shake out flaky tests like this.)

--- a/test/torch/tensors/test_precision.py
+++ b/test/torch/tensors/test_precision.py
@@ -461,7 +461,7 @@ def test_torch_exp_approx(prec_frac, tolerance, workers):
     [
         ("chebyshev", 3, 6 / 100),
         ("chebyshev", 4, 1 / 1000),
-        ("exp", 3, 6 / 100),
+        ("exp", 3, 6.5 / 100),
         ("exp", 4, 1 / 100),
         ("maclaurin", 3, 7 / 100),
         ("maclaurin", 4, 15 / 100),

